entries/t/ticketsource.co.uk.json

```
@@ -0,0 +1,15 @@
+{
+  "TicketSource": {
+    "domain": "ticketsource.co.uk",
+    "tfa": [
+      "totp"
+    ],
+    "keywords": [
+      "marketing",
+      "tickets"
+    ],
+    "regions": [
+      "gb"

```

```suggestion
      "gb",
      "be",
      "bg",
      "cz",
      "dk",
      "de",
      "ee",
      "ie",
      "gr",
      "es",
      "fr",
      "hr",
      "it",
      "cy",
      "lv",
      "lt",
      "lu",
      "hu",
      "mt",
      "nl",
      "at",
      "pl",
      "pt",
      "ro",
      "si",
      "sk",
      "fi",
      "se",
      "us"
```

TicketSource is available in the EU and the U.S.